src/sagemaker/workflow/step_collections.py

```diff
@@ -400,6 +400,7 @@ def __init__(
                 security_group_ids=estimator.security_group_ids,
                 description=description,
                 display_name=display_name,
+                output_kms_key=estimator.output_kms_key,
```

Comment:
Thanks so much for helping to fix the issue!

By adding this line, you're updating the `EstimatorTransformer` behavior. 
Can we make similar changes to the TWO `_RepackModelStep` objects created in `RegisterModel` as well? Thanks!

Reply:
Hi @qidewenwhen, I think you are already providing the output_kms_key in the Register Step as a kwarg. It was only kssing on the Transformer and in the ModelStep.

Can you please check it again?

Reply:
Yea you're right. It's already covered here https://github.com/aws/sagemaker-python-sdk/blob/849ed29a7f1ff9f1526b23902ad7db7c384126a9/src/sagemaker/workflow/step_collections.py#L171 in the `RegisterModel`.
Again appreciate your help!

Will reach out to @mufaddal-rohawala to review and merge the PR in Monday.